.buildkite/scripts/common.sh

```diff
@@ -92,7 +92,7 @@ google_cloud_auth() {
 
     gcloud auth activate-service-account --key-file ${keyFile} 2> /dev/null
 
-    export GOOGLE_APPLICATIONS_CREDENTIALS=${secretFileLocation}
```

Comment:
is there any `unset` command that needs to be updated along with this ?

Reply:
I'm sure yes, and something is already implemented here with the correct variable name.
Also, I'm not sure I have to change the logic of the pipeline. 
I just found the issue in my pipeline and suggested changing everywhere where it was copied because I spent some time to find and fix the issue.

Reply:
Ok, it was just to double check it was not left behind some `unset` of this variable. In some repos, like elastic-package or fleet-server, those unset's are also in some trap functions or pre-exit hooks.

I just checked that there is just one usage of this env. var in this repo :+1: 
```shell
 $ git grep GOOGLE_APPLICATIONS
.buildkite/scripts/common.sh:    export GOOGLE_APPLICATIONS_CREDENTIALS=${secretFileLocation}
```